src/index.html

```diff
@@ -11,7 +11,148 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname">Surname:</label>
+        <input
+          cletype="text"
+          class="distance"
+          id="surname"
+          name="surname"
+          autocomplete="off"
+        ><br>
```

Comment:
Why do you need these `br` tags everywhere? Remove them
```suggestion
        >
```